src/helpers/utils.ts

```diff
@@ -0,0 +1,130 @@
+// import fs from "fs";
+const mappings = require("../../data/usda_rdi_nutrient_mapping.json");
+import { allAges, lifeStageItems } from "../RealData/realData";
+import { NutrientItem, NutritionFact } from "../models/pages";
+import { SelectedNutrient } from "../models/components";
+
+const ARTIFACT_PATH = ".raw";
+
+/**
+ * The purpose is to use `-` as separator in the URL naming scheme.
+ * This is important for SEO purposes
+ * Watch https://www.youtube.com/watch?v=AQcSFsQyct8 to learn more
+ * @param name
+ */
+export const spaceToDashes = (name: string) => {
+  const pathname = name
+    .toLowerCase()
+    .trim()
+    .replace(/[^\w]+/g, "-");
+  return pathname.endsWith("-") ? pathname.substr(0, pathname.length - 1) : pathname;
+};
+
+// export const writeJsonToFile = (fileName: string, jsonData: Object[]) => {
```

Comment:
We should enable this code and write index

Reply:
removed. Not needed